pkg/ocm/helpers.go

```diff
@@ -655,3 +656,77 @@ func ValidateAvailabilityZonesCount(multiAZ bool, availabilityZonesCount int) er
 
 	return nil
 }
+
+func (c *Client) GetVersion(userRequestedVersion string, channelGroup string) (string, error) {
```

Comment:
@gdbranco can we reuse the existing methods

Reply:
Current methods return versions with a different format, for the policy version the create/upgrade acc. roles command was already filtering for 'major.minor' format through custom methods within each file, but I decided to unify those in the helpers.go file just to make it more concise.